packages/rx_bloc_cli/mason_templates/bricks/feature_widget_toolkit/__brick__/lib/l10n/network_lookup/widget_toolkit_lookup.dart

```diff
@@ -22,9 +22,6 @@ class AppI18nWidgetToolkitLookup extends I18nFeatureWidgetToolkitLookup {
               I18nFeatureWidgetToolkit(I18nFeatureWidgetToolkitLookup_bg());
           break;
         case 'en':
-          bundledI18n =
```

Comment:
Please make sure such changes are addressed in separate PRs. From what I see this is not related to QR code integration, but feel free to correct me if I'm wrong